communities/microgalaxy/lab/sections/3_microbiome.yml

```diff
@@ -1,6 +1,25 @@
 id: microbiome
 title: Microbiome
 tabs:
+- id: overview
+  title: Overview
+  heading_md: >
+    Overview of microbial data for microbiome samples analysis tasks
+  content:
+    - image: "{{ figure_url }}"
+      caption_md: >
+        **Figure: Overview of microbial data analysis tasks for microbiome samples, including metabarcoding, metagenomics, metatranscriptomics, and metaproteomics, with corresponding tool suites and potential analyses available on microGalaxy.**  
+        The workflow progresses from raw data to intermediate and final outputs. Outputs are color-coded by type, with numbers indicating tool suites used for each step:  
+
+        - **Input data (blue):** Represent raw datasets, including sequencing reads and mass spectra.  
+        - **Intermediate outputs (gray):** Include cleaned reads, assemblies, and OTUs/AVSs.  
+        - **Taxonomic outputs (green):** Represent taxonomic profiling results.  
+        - **Functional outputs (yellow):** Highlight functional profiling and differential feature analysis.  
+        - **Other outputs (orange):** Indicate annotated genomes, reconstructed genomes, and antibiotic resistance genes.  
+
+        **Numbers correspond to tool suites:**  
+        1 (FastQC, Cutadapt, fastp, MultiQC), 2 (MetaPhlAn, Kraken, Diamond), 4 (Bakta, Prokka), 5 (HMMER), 6 (AMRFinderPlus, hAMRonization), 7 (ABRicate, PATRIC), 8 (ISEScan, Transit), 9 (StarAMR, MLST), 10 (CRISPRCasFinder), 11 (antiSMASH), 12 (Gene Ontology, KEGG), 14 (BWA, Bowtie), 21 (msconvert, OpenMS), 22 (SearchGUI, MaxQuant), 23 (PepQuery, UniPept), 24 (MSStats), 25 (QIIME2, Dada2), 26 (Picrust), 27 (MaAsLin2, edgeR), 28 (Krakentools), 29 (KRONA, Pavian), 30 (HUMAnN), 31 (Groot, deepARG), 32 (MEGAHIT, MetaSPAdes), 33 (MetaBAT, DASTool), 34 (GTDB-Tk), 35 (Metanovo), 36 (Unipept), 37 (MetaQuantome).
```

Comment:
Could you format as in the paper `(1) FastQC, etc`?

Reply:
the image is added, number are corrected however, I have a problem the only way to represent the image without syntax error I found until now is via a view button, which I don't like. I still need to figure another way

Reply:
I am trying html tags now I guess it will work